src/libs/page-detect.js

```diff
@@ -6,8 +6,13 @@ export const isDashboard = () => location.pathname === '/' || /^(\/orgs\/[^/]+)?
 
 export const isTrending = () => location.pathname.startsWith('/trending');
 
+export const isNotifications = () => /^\/(?:[^/]+\/[^/]+\/)?notifications/.test(location.pathname);
+
 // @todo Replace with DOM-based test because this is too generic #708
-export const isRepo = () => !isGist() && !isTrending() && /^\/[^/]+\/[^/]+/.test(location.pathname);
+export const isRepo = () => !isGist() &&
+														!isTrending() &&
+														!isNotifications() &&
+														/^\/[^/]+\/[^/]+/.test(location.pathname);
```

Comment:
A tad over indented, this only needs one tab

Reply:
I tried to visually align, but with tabs I guess it's impossible if there's no agreed `tabwidth`.

It's a pity, it looks substantially clearer with spaces (or if you set Tab == 2 spaces).

```js
export const isRepo = () => !isGist() &&
                            !isTrending() &&
                            !isNotifications() &&
                            /^\/[^/]+\/[^/]+/.test(location.pathname);
```

vs.

```js
export const isRepo = () => !isGist() &&
  !isTrending() &&
  !isNotifications() &&
  /^\/[^/]+\/[^/]+/.test(location.pathname);
```

I guess this is impossible with tabs?

Reply:
Yeah, with tabs there's always one per indentation change, never more. `xo` is configured to disallow all [kinds of alignments](https://github.com/sindresorhus/eslint-config-xo/blob/master/index.js#L88). I'm surprised that this passed the test at all.

If you prefer, putting the regex first would be a good compromise.

Reply:
Okay, done.